autogen/main/variables.tf.tmpl

```diff
@@ -645,33 +643,34 @@ variable "timeouts" {
     error_message = "Only create, update, delete timeouts can be specified."
   }
 }
-{% if beta_cluster %}
-  {% if autopilot_cluster != true %}
 
-variable "enable_kubernetes_alpha" {
+variable "monitoring_enable_managed_prometheus" {
```

Comment:
The logging and monitoring are pre-configured for autopilot, so these attributes would only apply for non-autopilot clusters

Reply:
https://cloud.google.com/kubernetes-engine/docs/resources/autopilot-standard-feature-comparison